packages/suite/src/hooks/wallet/form/useCompose.ts

```diff
@@ -114,6 +113,12 @@ export const useCompose = ({
                         type: 'compose',
                         message: errorMessage as any, // setError types is broken? according to ts it accepts only strings, but object or react component could be used as well...
                     });
+                } else if (defaultFieldRef.current !== 'outputs[0].amount') {
```

Comment:
just a nitpick, I would add comment explaining why is this needed, could be really confusing without having full picture.

Reply:
if you expand that block the you will see the whole picture :)
- set error to the field that triggered compose process
- or set error to default field (if it's not an amount, which is default-default)
- or set error to all present amount fields

Reply:
but ok, ill do the explanation here